azure_dashboard.cmake

```diff
@@ -45,8 +45,15 @@ if(NOT CTEST_BUILD_NAME)
   if(DEFINED ENV{SYSTEM_PULLREQUEST_SOURCEBRANCH})
     set(branch "-$ENV{SYSTEM_PULLREQUEST_SOURCEBRANCH}")
     set(dashboard_model "Experimental")
-  else()
+  elseif(ENV{BUILD_SOURCEBRANCHNAME} STREQUAL "master")
     set(branch "-master")
+    set(dashboard_model "Continuous")
+  elseif(ENV{BUILD_SOURCEBRANCHNAME} STREQUAL "nightly-master")
+    set(branch "-nightly-master")
+    set(dashboard_model "Nightly")
+  elseif(ENV{BUILD_SOURCEBRANCHNAME} STREQUAL "nightly-master")
```

Comment:
Should this just be `else()`?

Reply:
I think @blowekamp is correct that the last branch needs to be the catch all, and it is currently a copy of line 51.